client/app/app.js

```diff
@@ -0,0 +1,51 @@
+import { h, Component } from 'preact'
```

Comment:
It's a personal preference to really separate the JSX part from the pure JS. And we can have `a .jsx file = a (p)react component`. But maybe we can have others opinion. @gregorylegarec ? @m4dz ? or @GoOz ?